draft-ietf-quic-transport.md

```diff
@@ -2619,14 +2624,14 @@ At any time, endpoints MAY change the Destination Connection ID they transmit
 with to a value that has not been used on another path.
 
 An endpoint MUST NOT reuse a connection ID when sending from more than one local
-address, for example when initiating connection migration as described in
+address -- for example, when initiating connection migration as described in
 {{initiating-migration}} or when probing a new network path as described in
 {{probing}}.
 
 Similarly, an endpoint MUST NOT reuse a connection ID when sending to more than
 one destination address.  Due to network changes outside the control of its
 peer, an endpoint might receive packets from a new source address with the same
-destination connection ID, in which case it MAY continue to use the current
+Destination Connection ID, in which case it MAY continue to use the current
```

Comment:
This is about the value of the field, and should not be capitalized, right?

Reply:
We've capitalized field names, so I think that this is right.  We don't use "destination" without referring to the field.

Do you want to say "field value" here to further clarify?

Reply:
Good point. We do use `Destination Connection ID` elsewhere too, so I think this is ok.